packages/core/src/http-client.ts

```diff
@@ -77,13 +79,21 @@ export async function executeRequest(
     );
     return response;
   } catch (error: any) {
+    await handleStreamError(error);
```

Comment:
[pp] Add a todo reminder linking the [issue](https://github.com/axios/axios/issues/6468) to remove this once Axios provides a fix.